client/src/components/sidebar/friends/UserProfile.tsx

```diff
@@ -13,16 +13,37 @@ const StyledContainer = styled('div')`
 `;
 
 const StyledFullname = styled('div')`
-  word-break: break-all;
   font-size: 0.9rem;
+  white-space: nowrap;
 `;
 
-const getFullName = (firstName: string, lastName: string) => {
+const getTextWidth = (text: string, font: string): number => {
+  const canvas = document.createElement('canvas');
+
+  const context = canvas.getContext('2d');
+  if (!context) {
+    return 0;
+  }
+  context.font = font;
+  const metrics = context.measureText(text);
+  return metrics.width
+}
+
+const getFullName = (firstName:string, lastName: string) => {
```

Comment:
formatting fix (make sure to setup auto formatting in your vscode with prettier)
```suggestion
  return metrics.width;
};

const getFullName = (firstName: string, lastName: string) => {
```